crates/sui-core/src/epoch/epoch_store.rs

```diff
@@ -19,12 +20,21 @@ pub struct EpochStore {
 }
 
 impl EpochStore {
-    pub fn new(path: PathBuf) -> Self {
-        Self::open_tables_read_write(path, None)
+    pub fn new(path: PathBuf, genesis_committee: &Committee) -> Self {
+        let epoch_store = Self::open_tables_read_write(path, None);
```

Comment:
Is this None DB options? We should always allow callers to define DB options, so lets pass in an Option.